Publish update about the bridges (#2738)
matrix-org/matrix.org: content/blog/2025/05/2025-04-25-bridges-update.md

+++
date = "2025-05-14"
title = "An Update on Bridges"

[taxonomies]
author = ["Thib"]
category = ["Bridges"]
+++

Following up on our [Crossroads blog post](/blog/2025/02/crossroads/) we’re giving an update on bridges. The Foundation indeed can’t afford to keep maintaining the bridges anymore and needs to focus its very little resources on its core missions. 

> Element manifested their interest in helping us keep the following public instances of the bridges online.

Reviewer: Let's be perfectly clear that this means all current bridges :tada: 
```suggestion
Element manifested their interest in helping us keep online *all* bridges we currently offer as public services. Specifically, this includes:
```

> | Instance | Platform | Project |

Reviewer: I'm trying to think of a better word than "project" but it doesn't come to mind. Bridge software seems too clunky.

> | Instance | Platform | Project |

Reviewer: I wish we could point to some docs about how to use the specific instance? We don't exactly have m.org hs docs on the site yet. Is there anything else suitable?

| -------- | -------- | ------- |
| OFTC     | IRC      | [matrix-appservice-irc](https://github.com/matrix-org/matrix-appservice-irc/) |
| Snoonet  | IRC      | [matrix-appservice-irc](https://github.com/matrix-org/matrix-appservice-irc/) |
| Slack    | Slack    | [matrix-appservice-slack](https://github.com/matrix-org/matrix-appservice-slack)  |
| Bifrost  | XMPP     | [matrix-bifrost](https://github.com/matrix-org/matrix-bifrost)  |

> We’re grateful to Element for sponsoring the bridges and keeping them online for the thousands of users who rely on them. Since Element already supports us in many areas, we keep actively looking for additional sponsors to help us grow more independent. You can find ways [to support us on our website](https://matrix.org/support/), the most impactful one being to [join us as a member](https://matrix.org/membership/).

Reviewer 2: ```suggestion
We’re grateful to Element for sponsoring the bridges and keeping them online for the thousands of users who rely on them. Since Element already supports us in many areas, we keep actively looking for additional sponsors to help us grow more independent. You can find ways [to support us on our website](/support/), the most impactful one being to [join us as a member](/membership/).
```

Relative links please :)

> In short: nothing changes from the user perspective, the bridges remain online. People, communities and organizations who want to deploy their own bridge regardless can do so by going to [https://matrix.org/ecosystem/bridges/](https://matrix.org/ecosystem/bridges/) and deploying their own instance.

Reviewer 2: ```suggestion
In short: nothing changes from the user perspective, the bridges remain online. People, communities and organizations who want to deploy their own bridge regardless can do so by going to [https://matrix.org/ecosystem/bridges/](/ecosystem/bridges/) and deploying their own instance.
```
